documentation/writing_plans.md

```diff
@@ -203,133 +203,8 @@ automatically masked.
 
 ## Using Hiera data in plans
```

Comment:
Should this just be a `Related information` somewhere on this page? Or at least not an H2?

Reply:
I kind of went back and forth on this and ended up just deciding it was okay, but now that you've brought it up as well, I think we should change it to a related link :D

Reply:
Added a "Related information" link under the intro section for the page.